examples/cookiejar/cookiejar.go

```diff
@@ -0,0 +1,59 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+	"net/http/cookiejar"
+	"net/url"
+	"time"
+
+	"github.com/qlik-oss/enigma-go"
+)
+
+func main() {
+
+	// Create dialer
+	dialer := enigma.Dialer{}
+
+	//Add a cookiejar
+	jar, err := dummyCookieJar()
+	if err != nil {
+		fmt.Print(err)
+	}
+
+	//Set dialer jar
+	dialer.Jar = jar
+
+	//Print all session cookies with the URL "https://www.qlik.com/"
+	urlTest, err := url.Parse("https://www.qlik.com/")
+
+	cookies := dialer.Jar.Cookies(urlTest)
+	for _, cookie := range cookies {
+		if cookie.Name == "_session" {
+			fmt.Println(cookie)
```

Comment:
I think it would be nice if there was some additional information for context.
Maybe something as simple as "Cookies found:" before the cookies are printed.